scripts/config.ts

```diff
@@ -17,7 +17,7 @@ export const ASSET_OBJECT_MAP: {[key: string]: string} = {
   'shopper-customers': 'a003k00000UHvpJAAT',
   'shopper-discovery-search': 'a003k00000W0WlBAAV',
   'shopper-gift-certificates': 'a003k00000UHvogAAD',
-  'shopper-login': 'a003k00000VWfNDAA1',
+  'shopper-login-and-api-access': 'a003k00000VWfNDAA1',
```

Comment:
Got it! The reason I changed this is because the api docs page for shopper-login has shopper-login-and-api-access extension. I didn't know a better way to redirect to the right page for the login docs page.